dist/obs-server.spec

```diff
@@ -492,6 +496,12 @@ make
 %sysusers_generate_pre dist/system-user-obsservicerun.conf obsservicerun system-user-obsservicerun.conf
 %endif
 
+# combine swagger yaml files to one big yaml file by resolving all references
+# and replace the development version
+pushd /dist
+ruby resolve_swagger_yaml.rb -i %{__obs_api_prefix}/public/apidocs-new/OBS-v2.10.50.yaml -o %{__obs_api_prefix}/public/apidocs-new/OBS-v2.10.50.yaml -f
+popd
```

Comment:
I guess you could do this in `dist/Makefile` but it's okay for me to do this later...